deploy/ui/form.json

```diff
@@ -19,33 +19,56 @@
           "originalJsonPath": "$.spec.environment",
           "type": "dropDown"
         },
+
         {
-          "label": "Image Registry",
-          "default": "rhpam-trial",
+          "label": "External registry",
```

Comment:
Is there a technical restriction to using the same label of "Image Registry"?

Reply:
No, there isn't